vividus-plugin-web-app/src/main/java/org/vividus/bdd/steps/ui/web/SetVariableSteps.java

```diff
@@ -122,6 +122,28 @@ public void savePathFromUrl(Set<VariableScope> scopes, String variable)
         saveVariable(scopes, variable, value);
     }
 
+    /**
+     * Gets a list of elements by <b>locator</b>
+     * and saves it into the <b>variable</b>
+     * @param scopes The set (comma separated list of scopes e.g.: STORY, NEXT_BATCHES) of variable's scope<br>
+     * <i>Available scopes:</i>
+     * <ul>
+     * <li><b>STEP</b> - the variable will be available only within the step,
+     * <li><b>SCENARIO</b> - the variable will be available only within the scenario,
+     * <li><b>STORY</b> - the variable will be available within the whole story,
+     * <li><b>NEXT_BATCHES</b> - the variable will be available starting from next batch
+     * </ul>
+     * @param locator to locate field
+     * @param variable A name under which the value should be saved
+     */
+    @When("I get list of element located `$locator` and set it to the $scopes variable '$variable'")
+    public void saveWebElements(SearchAttributes locator, Set<VariableScope> scopes, String variable)
+    {
+        List<WebElement> elements = baseValidations.assertIfElementsExist(String.format("A fields with attributes%1$s",
+                locator), locator);
+        saveVariable(scopes, variable, elements);
```

Comment:
we shouldn't save web elements, because we disclose internals to users and force them to understand the types